images/airflow/2.9.2/python/mwaa/celery/task_monitor.py

```diff
@@ -136,14 +147,16 @@ def _create_shared_mem_celery_state():
 # will use to signal the toggle of a flag which tells the Celery SQS channel to
 # pause/unpause further consumption of available SQS messages. It is maintained by the
 # worker monitor.
-def _create_shared_mem_work_consumption_block():
+# When MWAA signal handling is enabled, the consumption will be turned off by default and
```

Comment:
Maybe: `the consumption` -> `Airflow Task consumption`

Reply:
Sure